mappings/net/minecraft/server/ServerConfigurationManager.mapping

```diff
@@ -57,6 +57,14 @@ CLASS none/mr net/minecraft/server/ServerConfigurationManager
 		ARG 1 gameProfile
 	METHOD a getPlayer (Ljava/util/UUID;)Lnone/lw;
 		ARG 0 uuid
+	METHOD a sendToAllPlayersInRange (Lnone/aam;DDDDILnone/fl;)V
+		ARG 0 except
```

Comment:
It makes more sense to me since that's what it does. If `except` is non-null, it will not send it to that player.